lib/sequent/test/command_handler_helpers.rb

```diff
@@ -36,45 +36,56 @@ module CommandHandlerHelpers
 
       class FakeEventStore
         def initialize
-          @all_events = []
+          @event_streams = {}
+          @all_events = {}
           @stored_events = []
         end
 
         def load_events(aggregate_id)
-          deserialize_events(@all_events).select { |event| aggregate_id == event.aggregate_id }
+          event_stream = @event_streams[aggregate_id]
+          return nil unless event_stream
```

Comment:
This method returns `Option[(StreamRecord, Array[Event])]` so an empty stream does not make sense (there is no `StreamRecord` to return).